prov/sockets/src/sock_ep.c

```diff
@@ -1790,7 +1793,8 @@ int sock_alloc_endpoint(struct fid_domain *domain, struct fi_info *info,
 	}
 
 	if (sock_conn_map_init(sock_ep, sock_cm_def_map_sz)) {
-		SOCK_LOG_ERROR("failed to init connection map: %s\n", strerror(errno));
+		SOCK_LOG_ERROR("failed to init connection map: %s\n",
+			       strerror(ofi_sockerr()));
```

Comment:
If sockerr is needed, it should be pushed closer to the call inside sock_conn_map_init() where required.

Reply:
I definitely agree with you. Will hide specific to `fi_epoll_*` routines.